Homeworks/01-concurrent-matrix-multiplication/MatrixBenchmark/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿// <copyright file="Program.cs" company="ArtemNikit1n">
+// Licensed under the MIT License. See LICENSE file in the project root for full license information.
+// </copyright>
+
+using ConcurrentMatrixMultiplication;
+
+if (args.Length >= 3)
```

Comment:
```suggestion
if (args.Length < 3)
{
    Console.WriteLine("Usage: dotnet run -- <left_matrix> <right_matrix> <result> [--sequential|--parallel]");
    Console.WriteLine("Examples:");
    Console.WriteLine("  dotnet run -- matrix1.txt matrix2.txt result.txt --parallel");
    Console.WriteLine("  dotnet run -- matrix1.txt matrix2.txt result.txt --sequential");
    return;
}
```
Лучше так, чтобы не создавать лишней вложенности